sources/packages/web/src/services/http/ConfirmationOfEnrollmentApi.ts

```diff
@@ -1,115 +1,128 @@
     );
-
-    // Search criteria is populated only when search box has search text in it.
-    if (paginationOptions.searchCriteria) {
-      url = `${url}&${PaginationParams.SearchCriteria}=${paginationOptions.searchCriteria}`;
-    }
-    return this.getCallTyped<PaginatedResults<COESummaryDTO>>(url);
   }
 
-  public async getApplicationForCOE(
+  /**
+   * Get the application details for the Confirmation Of Enrollment(COE).
+   * @param locationId location id.
+   * @param disbursementScheduleId disbursement schedule id of COE.
+   * @returns application details for COE.
+   */
+  async getApplicationForCOE(
     disbursementScheduleId: number,
     locationId: number,
-  ): Promise<ApplicationDetailsForCOEDTO> {
-    try {
-      const response = await this.apiClient.get(
-        `institution/location/${locationId}/confirmation-of-enrollment/disbursement/${disbursementScheduleId}`,
-        this.addAuthHeader(),
-      );
-      return response.data;
-    } catch (error) {
-      this.handleRequestError(error);
-      throw error;
-    }
+  ): Promise<ApplicationDetailsForCOEAPIOutDTO> {
+    return this.getCallTyped<ApplicationDetailsForCOEAPIOutDTO>(
+      this.addClientRoot(
+        `location/${locationId}/confirmation-of-enrollment/disbursement-schedule/${disbursementScheduleId}`,
+      ),
+    );
   }
 
-  public async confirmCOE(
+  /**
+   * Approve confirmation of enrollment(COE).
+   * An application can have up to two COEs based on the disbursement schedule,
+   * hence the COE approval happens twice for application with more than once disbursement.
+   * Irrespective of number of COEs to be approved, application status is set to complete
+   * on first COE approval.
+   * @param locationId location id of the application.
+   * @param disbursementScheduleId disbursement schedule id of COE.
+   * @param payload COE confirmation information.
+   */
+  async confirmEnrollment(
     locationId: number,
     disbursementScheduleId: number,
-    confirmationData: ConfirmationOfEnrollmentAPIInDTO,
+    payload: ConfirmationOfEnrollmentAPIInDTO,
   ): Promise<void> {
     try {
       await this.patchCall(
-        `institution/location/${locationId}/confirmation-of-enrollment/disbursement/${disbursementScheduleId}/confirm`,
-        confirmationData,
+        this.addClientRoot(
+          `location/${locationId}/confirmation-of-enrollment/disbursement-schedule/${disbursementScheduleId}/confirm`,
+        ),
+        payload,
       );
     } catch (error: unknown) {
       this.handleAPICustomError(error);
     }
   }
 
-  public async rollbackCOE(
+  /**
+   * Creates a new Student Application to maintain history,
+   * overriding the current one in order to rollback the
+   * process and start the assessment all over again.
+   * @param locationId location id executing the COE rollback.
+   * @param disbursementScheduleId application to be rolled back.
+   * @returns the id of the newly created Student Application.
```

Comment:
the comment looks wrong. no need of `returns` comment

Reply:
I will remove from both methods Api/Service.